themes/gauge_theme/layout.html

```diff
@@ -164,37 +146,63 @@
 {%- endblock %}
 
 {%- block footer %}
-  <footer>
-    <div class="powered-by">
-      <div class="container">
-        <a href="https://www.thoughtworks.com/products">
-          <img alt="Thoughtworks logo" src="{{ pathto('_static/img/thoughtworks-logo.svg', 1) }}" height="44">
-        </a>
-        <div><a href="browse.html">Table of Contents</a></div>
-        <div class="disclamer">
-          Gauge is an open source project, sponsored by ThoughtWorks Inc. under the GPL License, Version 3.0
+<footer>
+  <div class="footer container">
+    <div class="footer_items product">
+      <h5>Product</h5>
+      <nav class="footer_links">
+        <ul class="foolinks">
+          <li> <a href="get_started/">Get started</a> </li>
+          <li> <a href="/index.html#features">Features</a> </li>
+          <li> <a href="plugins/">Plugins</a> </li>
+          <li> <a href="https://docs.gauge.org/latest/index.html" target="_blank">Documentation</a> </li>
+          <li> <a href="https://blog.getgauge.io/"  target="_blank">Blog</a></li> 
+          <li> <a href="#https://brandfolder.com/twproducts"   target="_blank">Press Kit</a></li> 
```

Comment:
Please remove the `#` at the start of the `href`
Also change all urls to `'://` instead of `https://`